docs/releases/README.md

```diff
@@ -12,13 +12,25 @@ Automatic development / test / production builds are still a work in progress, b
 
 - `+dirty` for uncommited changes
 - `-version` flag that can be injected or defaults to `UNKNOWN`
-- `branch_name` and a shortened `commit_hash` shold be included
+- `branch_name` and a shortened `commit_hash` should be included
 
 For example, `X.Y.Z[-<pre_release_tag][+branch_name][+short_hash][+dirty]` is the name of a potential build we will release in the future.
 
-## Supplied artifacts
+## Container Images
+
+### Tags
```

Comment:
Can you insert links to the images built for reference for each of the subsections below?

Reply:
@Olshansk instead of attaching an image, I've put a link to the page with list of tags which can be used as a reference. Note, this container repo is currently private, and we should make it public.

Reply:
Nice!